cmd/kpod/images.go

```diff
@@ -55,155 +84,250 @@ func imagesCmd(c *cli.Context) error {
 	if err := validateFlags(c, imagesFlags); err != nil {
 		return err
 	}
-	config, err := getConfig(c)
-	if err != nil {
-		return errors.Wrapf(err, "Could not get config")
-	}
-	store, err := getStore(config)
+
+	runtime, err := getRuntime(c)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "Could not get runtime")
 	}
+	defer runtime.Shutdown(false)
 
-	quiet := false
-	if c.IsSet("quiet") {
-		quiet = c.Bool("quiet")
-	}
-	noheading := false
-	if c.IsSet("noheading") {
-		noheading = c.Bool("noheading")
-	}
-	truncate := true
-	if c.IsSet("no-trunc") {
-		truncate = !c.Bool("no-trunc")
-	}
-	digests := false
-	if c.IsSet("digests") {
-		digests = c.Bool("digests")
-	}
-	outputFormat := genImagesFormat(quiet, truncate, digests)
+	var format string
 	if c.IsSet("format") {
-		outputFormat = c.String("format")
+		format = c.String("format")
+	} else {
+		format = genImagesFormat(c.Bool("quiet"), c.Bool("noheading"), c.Bool("digests"))
+	}
+
+	opts := imagesOptions{
+		quiet:     c.Bool("quiet"),
+		noHeading: c.Bool("noheading"),
+		noTrunc:   c.Bool("no-trunc"),
+		digests:   c.Bool("digests"),
+		format:    format,
 	}
 
-	name := ""
+	imageInput := ""
 	if len(c.Args()) == 1 {
-		name = c.Args().Get(0)
+		imageInput = c.Args().Get(0)
 	} else if len(c.Args()) > 1 {
 		return errors.New("'kpod images' requires at most 1 argument")
 	}
 
-	var params *libpod.FilterParams
+	filter := ""
 	if c.IsSet("filter") {
-		params, err = libpod.ParseFilter(store, c.String("filter"))
-		if err != nil {
-			return errors.Wrapf(err, "error parsing filter")
-		}
-	} else {
-		params = nil
+		filter = c.String("filter")
```

Comment:
This code block should just be
filter := c.String("filter")

Reply:
fixed